modules/rss-ticker.xml

```diff
@@ -226,7 +226,7 @@ if ((properties.takeItemsFrom === 'end' && properties.reverseOrder === 0) ||
         items = $(newItems);
     }
 }
-return items;
+return {dataItems: items};
```

Comment:
`calendar.xml` also has a `onDataLoad` that returns items, so before merging this PR to develop, you might want to merge develop to you local and add that